src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/finallocalvariable/Example1.java

```diff
@@ -0,0 +1,32 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="FinalLocalVariable"/>
+  </module>
+</module>
+*/
+
+
+
+
+
+
+package com.puppycrawl.tools.checkstyle.checks.coding.finallocalvariable;
+
+// xdoc section -- start
+class Example1
+{
+  static int foo(int x, int y) {
+    //ok, because PARAMETER_DEF is not configured in tokens
+    return x+y;
+  }
+  public static void main (String []args) {
+    //ok above, because PARAMETER_DEF is false by default
```

Comment:
```suggestion
  public static void main (String []args) {
    //ok above, because PARAMETER_DEF is not configured in tokens
```

Reply:
done

Reply:
Oh my bad forgot to update in the ``.xml`` template that's why the CI is failing